test/ref_ops_test.cpp

```diff
@@ -2687,6 +2688,58 @@ TEST_CASE(mul_test)
     EXPECT(migraphx::verify_range(results_vector, gold));
 }
 
+TEST_CASE(multinomial_test)
```

Comment:
Can you turn on the multinomimal test in the file test/py/onnx_backend_test.py to check whether they can pass? Particularly for those with a see input.